kola/tests/crio/crio.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2018 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package crio
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+	"time"
+
+	"golang.org/x/crypto/ssh"
+	"golang.org/x/net/context"
+
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/lang/worker"
+	"github.com/coreos/mantle/platform"
+)
+
+// simplifiedCrioInfo represents the results from crio info
+type simplifiedCrioInfo struct {
+	StorageDriver string `json:"storage_driver"`
+	StorageRoot   string `json:"storage_root"`
+	CgroupDriver  string `json:"cgroup_driver"`
+}
+
+// init runs when the package is imported and takes care of registering tests
+func init() {
+	register.Register(&register.Test{
+		Run:         crioBaseTests,
+		ClusterSize: 1,
+		Name:        `crio.base`,
+	})
+	register.Register(&register.Test{
+		Run:         crioNetwork,
+		ClusterSize: 2,
+		Name:        "crio.network",
```

Comment:
Probably should add `Distros:     []string{"rhcos"},` to each of these tests, so they will be limited to RHCOS

Would require #874

Reply:
Added as a comment.